src/BooksApi/BooksApi.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/NmerLOYOolwhXAVjaTow/books';
+
+export const fetchBooks = async () => {
+  const response = await axios.get(`${BASE_URL}/books`);
+  return response.data;
+};
+
+export const addBook = async (book) => {
+  const response = await axios.post(`${BASE_URL}/books`, book);
+  return response.data;
```

Comment:
* I get this error in the console when I try to add a book, Kindly fix it as the API doesn't function like it should 
![image](https://github.com/theresetuyi/React-Bookstore/assets/52497246/63fd0698-2ed0-495d-a6b7-95578c54adc2)